src/python/turicreate/test/test_style_transfer.py

```diff
@@ -292,6 +292,17 @@ def test_save_and_load(self):
             self.test_get_styles_success()
             print("Get styles passed")
 
+    def test_state(self):
+        model = self.model
+        fields = model.__proxy__.list_fields()
+        assert "model" in fields
+        assert "num_styles" in fields
+        assert "_training_time_as_string" in fields
+        assert "training_epochs" in fields
+        assert "training_iterations" in fields
+        assert "num_content_images" in fields
+        assert "training_loss" in fields
```

Comment:
Are we expecting other keys might in `fields`? If not, it would be good to check there aren't any other thing in there.

Reply:
We are expecting there to be other keys.